apps/AnyDesk/install-64

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+version=6.3.0-1
+
+function check-armhf() {
+	ARMHF="$(dpkg --print-foreign-architectures | grep "armhf")"
+}
+
+#add armhf architecture (multiarch)
+check-armhf
+if [[ "$ARMHF" == *"armhf"* ]]; then
+  echo "armhf arcitecture already added..."
+else
+  sudo dpkg --add-architecture armhf
+  check-armhf
+  if [[ "$ARMHF" != *"armhf"* ]]; then
+    error "armhf architecture should be added by now, but it isn't!"
+  fi
+fi
+
+unset rpi_arm_userspace
+# only install the libraspberrypi0 arm32 package if the user already has the libraspberrypi0 arm64 package installed
+if package_installed libraspberrypi0 ; then
+  rpi_arm_userspace="libraspberrypi0:armhf"
+fi
+
+install_packages https://download.anydesk.com/rpi/anydesk_${version}_armhf.deb libgles-dev:armhf libegl-dev:armhf libpolkit-gobject-1-0:armhf $rpi_arm_userspace patchelf || exit 1
```

Comment:
likely unnecessary dependencies
development libraries are nearly never required for a package
control file for anydesk already pulls in egl and glvnd libraries which pull in everything else needed

Reply:
these are the libs installed by libraspberrypi0:armhf:
```
/usr/lib/arm-linux-gnueabihf/libbcm_host.so.0
/usr/lib/arm-linux-gnueabihf/libdebug_sym.so.0
/usr/lib/arm-linux-gnueabihf/libdtovl.so.0
/usr/lib/arm-linux-gnueabihf/libvchiq_arm.so.0
/usr/lib/arm-linux-gnueabihf/libvcos.so.0
```
I am not sure that the default deps pull those in, at least for armhf

Reply:
I am referring to `libgles-dev:armhf libegl-dev:armhf`

Reply:
Without those 2 installed, and then reinstalling the anydesk package and allowing it to pull in deps:
```
$ anydesk
anydesk: error while loading shared libraries: libGLESv2.so: cannot open shared object file: No such file or directory
```

Reply:
Oh, I get it. AnyDesk should be able to use `libGLESv2.so.2` provided by libgles2, instead of specifically needing libGLESv2.so that is only provided by the -dev package.
Looks like this can be fixed with more elf patching.

Reply:
hmm... odd
```
/usr/lib/arm-linux-gnueabihf/libGLESv2.so	[libgles-dev](https://packages.debian.org/bookworm/libgles-dev) [armhf]
```
its not normal that a package uses this library.

those dependencies need to stay then

Reply:
@Botspot you missed my response ^
you need to revert those -dev changes from https://github.com/Botspot/pi-apps/pull/2455/commits/9b0ce0a37b64a196613866bd7da0af7f2c109288 . they are needed. I checked

Reply:
Is patching the elf to use the non-dev libs not sufficient?